### PR TITLE
Update Stepper docs

### PR DESCRIPTION
**Pull request by TVincke98**

### CODE REVIEW

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ To configure the component:
 
    - When a DOM instance is applied, this will be used to display the current state, preceded by the states the instance has traversed until its current state, and followed by the future states it will go through if it follows the normal flow.
 
+     > [!NOTE]
+     > Not all DOM instances keep a history of the states they traversed, this behaviour is decided by the module. When an instance has no history, the path from the first state to the current state is calculated in a similar way as the happy path after the current state. Concretely, this means you'll see the states the instance traverse through to get from the first possible state to the current state if it were to follow the normal flow.
```
Comment (WoutMH) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md]: reading this, it seems to me that for some instances in a module there is history and for others not, which is not the case. Maybe explain that it is a setting on the module that can be enabled/disabled (with a link to those docs)

**Review by EdithLansens — APPROVED** [review]
I have slightly reworded your note to make it as concise and clear as possible. Let me know if this is OK to merge or if something should be adjusted, @TVincke98.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ To configure the component:
 
    - When a DOM instance is applied, this will be used to display the current state, preceded by the states the instance has traversed until its current state, and followed by the future states it will go through if it follows the normal flow.
 
+     > [!NOTE]
+     >
+     > - Not all DOM instances maintain a history of their traversed states. This behavior is determined by the module. Instances without a history calculate the path from the initial state to the current state as if they followed the standard flow, similar to a happy path.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md:26]:
```suggestion
     > - If the DOM history saving behavior of a module has been set to *Disabled* in [the `DomInstanceHistorySettings` object](xref:DOM_DomInstanceHistorySettings), the stepper component will not show a history of traversed states for those DOM instances. Instances without a history calculate the path from the initial state to the current state as if they followed the standard flow, similar to a happy path.
```

Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md:26]: Would this be more accurate, @WoutMH?

Comment (WoutMH) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md:26]: Yes, looks good to me

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md [hunk 3]
```diff
@@ -21,6 +21,11 @@ To configure the component:
...
+     > - The [DOM instance history objects](xref:DOM_DomInstanceHistorySettings) are by default stored asynchronously, which enhances performance but may result in an outdated history when retrieved. In such cases, the path between the incomplete history and the current state is also completed according to the standard flow.
```
Comment (TVincke98) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md:27]: Maybe we can remove the hyperlink here then as it refers to the same documentation

Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md]: Done!

**Pull request merged**